src/components/Gradebook/index.jsx

```diff
@@ -683,7 +683,7 @@ export default class Gradebook extends React.Component {
                     options={[{ label: 'Percent', value: 'percent' }, { label: 'Absolute', value: 'absolute' }]}
                     onChange={this.props.toggleFormat}
                   />
-                  {this.props.showDownloadButtons && (
+                  {this.props.showBulkManagement && (
```

Comment:
Collapsing these two props into one, since it's clear they won't change separately in the future. (This wasn't as clear before this change was request.)